core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -211,6 +211,7 @@
 import static org.apache.calcite.sql.fun.SqlLibraryOperators.LOG2;
 import static org.apache.calcite.sql.fun.SqlLibraryOperators.LOGICAL_AND;
 import static org.apache.calcite.sql.fun.SqlLibraryOperators.LOGICAL_OR;
+import static org.apache.calcite.sql.fun.SqlLibraryOperators.LOG_MYSQLSPARK;
```

Comment:
It is strange we use LOG_MYSQLSPARK. May be LOG_SPARK or LOG_MYSQL. we will add the `libraries` label in SqlLibraryOperators to show the function can work in which database.

Reply:
Is LOG_MYSQL OK?

Reply:
Bigquery's log function and MySQL's log function are inconsistent in log(0)